server/views/pages/licence/HDC_AP.njk

```diff
@@ -61,7 +55,25 @@
    </div>
 
    {% include "pages/licence/partials/apDates.njk" %}
-   {% include "pages/licence/partials/induction.njk" %}
+   
+   <div id="induction" class="section">
+      <p>
+         On release from prison (including, if applicable, any release from detention under the Immigration Act 1971
+         during the currency of your licence, whether or not leave has been granted for you to remain in the United Kingdom),
+         unless otherwise directed by your supervising officer, you must report without delay to:
+      </p>
+      <div id="meeting-details">
+         <p>
+               <span data-qa="appointment-person" class="bold line-space-after">{{ appointmentPerson }}</span>
+               {% for a in (licence.appointmentAddress | formatAddressAsList) %}
+                  <span class="bold line-space-after">{{ a }}</span>
+               {% endfor %}
+         </p>
+         <p data-qa="appointment-time">On <span class="bold">{{ licence.appointmentTime | datetimeToDateWithDay }}</span> at <span class="bold">{{ licence.appointmentTime | datetimeTo12HourTime }}</span></p>
+         <p>Contact telephone number: <span class="bold">{{ licence.appointmentContact }}</span></p>
+      </div>
+   </div>
+
```

Comment:
Policy don’t want us to use options like “immediately after release” and "By 2pm next working day" that we use in CVL in AFER so removed this - think we will therefore need to adjust CVL to only show these options if HDC kind

Reply:
If there's not already a ticket for this, we should probably add one!
Wouldn't want it to slip through the cracks

Reply:
I've added a comment re this to the Allow creation of HDC cases in CVL ticket